### DISCUSSION TITLE
Repeater with multi-level relationship

### DISCUSSION THREAD

**Discussion by AbhinavNmdo** [filamentphp/filament#8006] [Help]
I have three tables `order_master`, `order_details`, `product_masters`.

`order_master` -> `hasMany` -> `order_details`
`order_details` -> `belongsTo` -> `product_masters`

Now, I want to add repeater in `OrderMasterResource` of order_masters with the relationship of `order_details` and add a field of product name in repeater which is belongs to `product_masters`.

Is there any way  to show this??

Thanks in Advance 😊😊

**Comment by micraux** [comment]
This is the next step in my actual project!
So I can't help you right now but will be able to do so by the weekend (if still needed)...

I suppose your `order_master` -> `belongsTo` -> `client` as well?
If it's the case (like me), you might find the Relation Manager feature helpful:
https://filamentphp.com/docs/3.x/panels/resources/relation-managers
